package.json

```
@@ -11,39 +11,40 @@
     },
     "private": true,
     "dependencies": {
-        "@angular/animations": "~9.0.5",
-        "@angular/cdk": "^9.1.1",
-        "@angular/common": "~9.0.5",
-        "@angular/compiler": "~9.0.5",
-        "@angular/core": "~9.0.5",
-        "@angular/forms": "~9.0.5",
-        "@angular/material": "^9.1.1",
-        "@angular/platform-browser": "~9.0.5",
-        "@angular/platform-browser-dynamic": "~9.0.5",
-        "@angular/router": "~9.0.5",
-        "rxjs": "~6.5.4",
-        "tslib": "^1.10.0",
-        "zone.js": "~0.10.2"
+        "@angular/animations": "~9.x.x",
```

For the '~' only the latest patch version is used, the minor version should still be defined:
 ```suggestion
        "@angular/animations": "~9.0.x",
```